pkg/discovery/endpoint/restapi/operations.go

```diff
@@ -201,39 +213,74 @@ func (o *Operation) writeResponseForResourceRequest(rw http.ResponseWriter, reso
 		writeResponse(rw, resp, http.StatusOK)
 	case strings.HasPrefix(resource, fmt.Sprintf("%s%s", o.baseURL, o.webCASPath)):
 		o.handleWebCASQuery(rw, resource)
-	case strings.HasPrefix(resource, fmt.Sprintf("%s%s", o.baseURL, "/services/orb")):
-		o.handleWitnessQuery(rw, resource)
-	case strings.HasPrefix(resource, fmt.Sprintf("%s%s", o.baseURL, "/anchor")):
-		o.handleAnchorQuery(rw, resource)
-	case strings.HasPrefix(resource, fmt.Sprintf("%s%s", o.baseURL, "/origin")):
-		o.handleOriginQuery(rw, resource)
-	case resource == o.baseURL:
-		resp := &WebFingerResponse{
-			Subject: o.baseURL,
+	case strings.HasPrefix(resource, "did:orb:"):
```

Comment:
As mentioned earlier this code should be moved to resource registry that will return JRD instead of map and handle all internals per resource (this is separate PR)

Reply:
Added to https://github.com/trustbloc/orb/issues/536#issuecomment-871016266.

TODO is on line 220